demo/loadmore.html

```diff
@@ -0,0 +1,88 @@
+<html>
+
+<head>
+  <title>Load more</title>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+
+  <style>
+    body {
+      font-family: Roboto, Helvetica, sans-serif;
+      max-width: 600px;
+      height: 100vh;
+      margin: 0px auto;
+      display: flex;
+      flex-direction: column;
+    }
+
+    virtual-scroller {
+      flex: 1;
+      margin: 10px 0;
+      border-top: 2px solid #eee;
+      border-bottom: 2px solid #eee;
+    }
+
+    contact-element,
+    button {
+      height: 80px;
+    }
+  </style>
+</head>
+
+<body>
+  <virtual-scroller></virtual-scroller>
+
+  <script type="module">
+    import '../virtual-scroller-element.js';
+    import './contacts/contact-element.js';
+    import { ContactDataSource } from './contacts/contact-data-source.js';
+
+    const dataSource = new ContactDataSource();
+    const virtualScroller = document.querySelector('virtual-scroller');
+
+    const button = document.createElement('button');
+    const loadMore = async () => {
+      // Start loading the contacts...
+      const promise = dataSource.getContacts(20);
+      // ... notify list so the button can be updated...
+      virtualScroller.itemsChanged();
```

Comment:
Would it work to move the updating code (setting disabled/textContent) to here? This feels a bit indirect.